website/sidebars.js

```diff
@@ -47,6 +47,7 @@ module.exports = {
         'developer/deployment-guide/deployment-environment',
         'developer/deployment-guide/deployment-s3-bucket',
         'developer/deployment-guide/deployment-with-cumulus',
+        'developer/deployment-guide/upgrading-orca'
```

Comment:
path should be `'developer/deployment-guide/deployment-upgrading-orca'` based on your page `id`.

Reply:
made update to sidebars.js.